src/components/Input.js

```diff
@@ -0,0 +1,230 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classnames from 'classnames';
+import defaultTheme from '../themes/DefaultTheme';
+import darkTheme from '../themes/DarkTheme';
+import { ErrorIcon } from './icons';
+
+class Input extends React.PureComponent {
+  constructor(props) {
+    super(props);
+  }
+
+  render() {
+    const {
+      name,
+      className,
+      type,
+      value,
+      placeholder,
+      label,
+      sublabel,
+      info,
+      error,
+      autoFocus,
+      readOnly,
+      disabled,
+      onFocus,
+      onKeyPress,
+      onChange,
+    } = this.props;
+
+    return (
+      <div className="drui-inputContainer">
+        {label && <span className="drui-input-label">{label}</span>}
+        {sublabel && <span className="drui-input-sublabel">{sublabel}</span>}
+        <div className="drui-inputIconContainer">
+          <input
+            name={name}
+            className={classnames('drui-input', {
+              'drui-input--disabled': disabled,
+              'drui-input--error': error,
+              'drui-input--readOnly': readOnly,
+              [className]: className,
+            })}
+            type={type}
+            value={value || undefined}
+            placeholder={placeholder}
+            autoFocus={autoFocus}
+            readOnly={readOnly}
+            disabled={disabled}
+            onFocus={onFocus}
+            onKeyPress={onKeyPress}
+            onChange={onChange}
+          />
+          {error && <ErrorIcon className="drui-input-error__errorIcon" />}
+        </div>
+        {info && !error && <div className="drui-input-info"><span>{info}</span></div>}
```

Comment:
I'm not sure if the info element should disappear when some error occurs :/

Reply:
Im not sure either, this is how it looks in dashboard now.

Reply:
Please, let the info element to be displayed no matter if there is an error, or not ;)